templates/.rubocop.yml

```diff
@@ -7,50 +7,17 @@ Metrics/LineLength:
 Style/Documentation:
   Enabled: false
 
-Style/NonNilCheck:
-  IncludeSemanticChanges: true
-
-Naming/PredicateName:
-  Exclude:
-    - 'app/serializers/**/*'
-
-Metrics/BlockLength:
-  Exclude:
-  - config/initializers/simple_form_bootstrap.rb
-  - 'Rakefile'
-  - '**/*.rake'
-  - 'spec/**/*.rb'
-
-Naming/FileName:
-  Exclude:
-  - 'Gemfile'
-
 Style/FrozenStringLiteralComment:
   Enabled: false
-  
-Style/FormatStringToken:
-  Enabled: false
 
-Rails/LexicallyScopedActionFilter:
-  Enabled: false
```

Comment:
I think this rule makes a lot of sense.

Reply:
This rules brings you to write code like:
```
def show
end
```
which does nothing. since Rails doesn't require you to write ii if is empty. also more code = more bugs. in addition, to me could be considered dead code as well.

Reply:
Hm… right, you don't have to specify these actions in Rails. But do you like that? Should we really omit the one line `def show; end`? We're in MVC and I think it's just to much magic to omit the "C" of it (even more since we definitely have an "M" and a "V"). It's just that I personally find it very irritating, if there is **no** sign of an action (if you're lucky, there's a `before_filter :show` which indicated that something exists there).